meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -33,6 +33,11 @@ $divider-margin: 1.25rem;
     margin-bottom: $r-spacer;
 }
 
+.filter-bar--stacked.filter-bar--open {
+    position: absolute;
+    z-index: 2000;
+}
+
```

Comment:
is this the best way to do the overlay? feels like I may have oversimplified a bit?